src/luascript.cpp

```diff
@@ -11795,9 +11798,8 @@ int LuaScriptInterface::luaGuildCreate(lua_State* L)
 	// Guild(id)
 	uint32_t id = getNumber<uint32_t>(L, 2);
 
-	Guild* guild = g_game.getGuild(id);
-	if (guild) {
-		pushUserdata<Guild>(L, guild);
+	if (const auto guild = g_game.getGuild(id)) {
```

Comment:
this can be const auto&?

Reply:
This returns a `shared_ptr` so using reference or a copy doesn't matter

Reply:
well probably, because we rely on the compiler doing the job for us